geode-core/src/distributedTest/java/org/apache/geode/internal/cache/BucketCreationCrashRegressionTest.java

```diff
@@ -185,6 +186,7 @@ private Properties createLocatorConfig() {
     config.setProperty(ENABLE_CLUSTER_CONFIGURATION, "false");
     config.setProperty(ENABLE_NETWORK_PARTITION_DETECTION, "false");
     config.setProperty(USE_CLUSTER_CONFIGURATION, "false");
+    config.setProperty(DISABLE_AUTO_RECONNECT, "true");
```

Comment:
Why did you have to make this test change for a refactoring PR?

Reply:
As I recall, this test crashes a node and does not expect it to auto-reconnect.  That caused a failure in one of my test runs so I disabled the feature.